TombEngine/Specific/setup.cpp

```diff
@@ -45,254 +44,6 @@ void InitialiseGameFlags()
 	Camera.underwater = false;
 }
 
-void ObjectObjects()
```

Comment:
Why this subfunction was merged into parent one? This decreases readability, and also it is not related to this PR purpose.

Reply:
Was leftover when i was doing the objects folder with every objects files.